odoo/addons/base/models/ir_model.py

```diff
@@ -994,6 +994,13 @@ def _reflect_field_params(self, field, model_id):
             'column2': field.column2 if field.type == 'many2many' else None,
         }
 
+    def _is_field_inherited_from_mixin(self, model, field_name):
```

Comment:
This does not need to be a method: it can be a function local to `_reflect_fields`.